wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+    <img src="../../../assets/img/pexels-dids-2508565.jpg">
```

Comment:
is this working if you build the angular application with ng build --prod?

Reply:
Yes, I think it does. Assets folder is inside the angular project, so I think it is  fine